proc_IR.py

```diff
@@ -19,25 +19,17 @@
         ('200212_IR_3_30dBm', 'test_equip', 'signal', 
             'spincore_IR_v1'),
         ]:
+    fl.basename = searchstr
```

Comment:
this will add searchstr to all titles and filenames

Reply:
this is cool! did you just add this? and should I incorporate this into the other processing scripts as well?